apis/storage.confluent.io/v1/localstorage_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+
+ */
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// LocalStorageSpec defines the desired state of LocalStorage
+type LocalStorageSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Foo is an example field of LocalStorage. Edit localstorage_types.go to remove/update
+	Replicas           int    `json:"replicas"`
+	InstanceType       string `json:"instanceType"`
+	ForceDeploy        bool   `json:"forceDeploy"`
+	EKSImage           string `json:"eksImage"`
+	NodeGrabberImage   string `json:"nodeGrabberImage"`
+	LocalVolumeImage   string `json:"localVolumeImage"`
+	StorageClassName   string `json:"storageClassName"`
+	ServiceAccountName string `json:"serviceAccountName"`
```

Comment:
The daemonsets require escalated permissions in order to create PVs and use local storage. In our case, we are adding these permissions to the operator controller's service account itself. If someone wants to use a different service account just for these resources then it would make sense.